lib/tasks/import_commitments.rake

```diff
@@ -35,8 +35,7 @@ namespace :import do
                                  country_ids: [country.id],
                                  committed_year: cbd_com["meta"]["createdOn"].to_date.year,
                                  update_year: cbd_com["meta"]["modifiedOn"].to_date.year,
-                                 related_biodiversity_targets: aichi_targets,
-                                 state: :live
```

Comment:
Should the old ones not be live?

Reply:
you're right, i've changed it so it now imports them as :draft, and then attempts to update them to :live. as they are now providing a new csv upload file to match the new  db, something will probably need to change here so the commitments imported via the cbd api are valid. e.g. they currently use Aichi targets rather than the objectives we are now going to use, so would fail the validations required to be 'live'